utils/utils.py

```diff
@@ -162,7 +185,7 @@ def _init():
         else:
             env = DummyVecEnv([_init])
     else:
-        env = DummyVecEnv([make_env(env_id, 0, seed, log_dir)])
+        env = DummyVecEnv([make_env(env_id, 0, seed, log_dir, wrapper_class=env_wrapper)])
```

Comment:
Why don't you wrap  also the env in the other cases ?

Reply:
I was afraid of touching atari and Bullet specific cases as I don't understand them well.
I wouldn't affect as long as no env_wrapper specified, though.
Let me add it for Bullet case but not for atari at least for now as it is using a special env constructor.

Reply:
ok, no worry, I think I'll do that in my PR with HER-support ;)